templates/etc_init_archivematica-storage-service.conf.j2

```diff
@@ -5,4 +5,8 @@ author "Artefactual Systems"
 start on (runlevel [2345] and filesystem)
 stop on runlevel [016]
 
+env LANG="en_US.UTF-8"
+env LC_ALL="en_US.UTF-8"
+env LC_LANG="en_US.UTF-8"
```

Comment:
Putting these env vars in `archivematica_src_ss_environment` in tasks/main.yml doesn't solve the issue for gunicorn deploys in my testing. If they are in etc_init_archivematica-storage-service.conf.j2 then you can create transfers with non-ascii characters in the transfer name. I don't know why this works and the other way doesn't, but it does ...